packages/odyssey/src/scss/base/_elements.scss

```diff
@@ -0,0 +1,86 @@
+blockquote {
+  padding: 0 $base-spacing;
+  border-left: 3px solid $base-border-color;
+}
+
+cite {
+  font-style: italic;
+}
+
+del {
+  display: inline-block;
+  background: cv('danger', 'lightest');
+}
+
+del::before,
+del::after {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+  clip: rect(1px, 1px, 1px, 1px);
+  white-space: nowrap;
+  clip-path: inset(100%);
+}
+
+del::before {
+  content: attr(data-a11y-start);
+}
+
+del::after {
+  content: attr(data-a11y-end);
+}
+
+em {
+  font-style: italic;
+
+  > em { text-decoration: underline; }
+}
+
+ins {
+  display: inline-block;
+  background: cv('success', 'lightest');
+}
+
+ins::before,
+ins::after {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+  clip: rect(1px, 1px, 1px, 1px);
+  white-space: nowrap;
+  clip-path: inset(100%);
+}
+
+ins::before {
+  content: attr(data-a11y-start);
+}
+
+ins::after {
+  content: attr(data-a11y-end);
+}
+
+mark { background: cv('warning', 'lightest'); }
+
+s { text-decoration: line-through; }
+
+sup {
+  font-size: 0.66012rem;
+  line-height: 1;
+  vertical-align: super;
+}
+
+sub {
+  font-size: 0.66012rem;
+  line-height: 1;
+  vertical-align: sub;
+}
+
+strong {
+  font-weight: bold;
```

Comment:
can we use `600` here? (we only ship 400 and 600 right now)

Reply:
fixed